www/settime

```diff
@@ -21,6 +22,7 @@ if ($db == false)
 // get the request parameters
 $game = get_req_data('game');
 $newtime = get_req_data('newtime');
+$timenote = get_req_data('note');
```

Comment:
So then should I change 
`$timenote = get_req_data('note');`
to this?
`$timenote = trim(get_req_data('note'));`

Reply:
Yeah, but it's not a high priority. Separate PR?